src/services/outliningElementsCollector.ts

```diff
@@ -113,21 +117,21 @@ namespace ts.OutliningElementsCollector {
                             parent.kind === SyntaxKind.WithStatement ||
                             parent.kind === SyntaxKind.CatchClause) {
 
-                            addOutliningSpan(parent, openBrace, closeBrace, autoCollapse(n));
+                            addOutliningSpan(parent, openBrace, closeBrace, autoCollapse(n), /* fullStart */ true);

```

Comment:
`s/fullStart/getFullStart`? Also, consider removing the spaces on either end of the comment to make it more compact. This *should* be the convention throughout the codebase -- it appears to be followed in the cases I saw within a cursory glance. Maybe a tslint rule could help here... @andy-ms ?

To avoid this in the future, we could make renaming smarter, but deciding how to handle comments in general is not obvious (c.f. csharp).